action/display.php

```diff
@@ -62,7 +62,7 @@ function handle_act_render(Doku_Event $event, $params) {
         if (!isset ($_REQUEST['linkback']) || $INFO['perm'] != AUTH_ADMIN)
             $_REQUEST['linkback'] = 'linkback_show';
 
-        $lid = $_REQUEST['lid'];
+        $lid = isset($_REQUEST['lid']) ? $_REQUEST['lid'] : '';
```

Comment:
General note, just for information: DokuWiki has a wrapper class available in `global $INPUT`, to clean up the request variables a bit and request them directly in the desired type. See https://www.dokuwiki.org/devel:request_vars 
Sometime it is simpler to use for these cases because it provides a default when the variable is not available.